spec/reference_entity.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+describe 'referenceEntity' do
+  before :all do
```

Comment:
No need to change this, but eventually I'd like to change this whole thing into something that does `let(:something) { Class.new(Grape::Entity) do ... }` this way we're not polluting global namespaces with custom names.